.travis.yml

```
@@ -1,10 +1,10 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
   - 7.0
-  - hhvm
+  - 7.1
+  - 7.2
+  - 7.3
```

PHP 7.4 is officially released so I think we should be able to support it here as well.

```suggestion
  - 7.3
  - 7.4
```